app/assets/javascripts/proxy_status.js

```diff
@@ -1,3 +1,8 @@
+$(document).on('ContentLoad', function() {
+  showProxies();
```

Comment:
if it gets triggered (besides on initial load) don't you need to set the spinner when its active?